playbooks/common/openshift-cluster/openshift_metrics.yml

```diff
@@ -1,14 +1,17 @@
 ---
-- name: OpenShift Metrics
+- name: Create Hosted Resources - metrics
   hosts: oo_first_master
   roles:
-  - openshift_metrics
+  - role: openshift_metrics
+    when: openshift_hosted_metrics_deploy | default(false) | bool
```

Comment:
Different variable is used after https://github.com/openshift/openshift-ansible/pull/5176

Reply:
`openshift_metrics_install_metrics`